### DISCUSSION TITLE
Support for Sublime

### DISCUSSION THREAD

**Discussion by noeltakacs** [olimorris/onedarkpro.nvim#253] [Ideas]
I believe that many developers, including myself, would greatly appreciate support for Sublime.

**Comment by olimorris** [comment]
Are you referring to Sublime Text?

**Reply by noeltakacs** [reply]
Yes

**Reply by olimorris** [reply]
I'd be surprised if any Sublime Text users are looking at a Neovim plugin but by all means please create an "Extra" template like I have with Alacritty and Kitty for example.
